### ISSUE TITLE
Implement webserver/interface

### ISSUE THREAD

**Issue by cgwalters** [cgwalters/coreos-diskimage-rehydrator#22]
It would make total sense to support being run as a webserver too, with a pretty interface.

One uncertainty is whether the API should appear just as a `GET` or require a `POST` to generate an image.  Generating some images is going to be sufficiently slow that I worry clients may time out on a `GET` request.

**Comment by ashcrow** [comment]
Agreed. I think a `PUT` or `POST` makes more sense for an API. Possibly respond back with a `uuid` which can be retrieved with a `GET` on progress.
